src/app/zap-templates/zcl/data-model/chip/joint-fabric-datastore-cluster.xml

```diff
@@ -0,0 +1,291 @@
+    <cluster code="0x0752"/>
+    <item fieldId="0" name="State" type="DatastoreStateEnum" default="Pending"/>
+    <item fieldId="1" name="UpdateTimestamp" type="epoch_us" isNullable="true"/>
+  </struct>
+
+  <struct name="DatastoreNodeKeyEntry">
+    <cluster code="0x0752"/>
+    <item fieldId="0" name="GroupKeySetId" type="int16u"/>
+    <item fieldId="1" name="StatusEntry" type="DatastoreStatusEntry"/>
+  </struct>
+
+  <struct name="DatastoreGroupInformationEntry" isFabricScoped="true">
+    <cluster code="0x0752"/>
+    <item fieldId="0" name="GroupID" type="int64u"/>
+    <item fieldId="1" name="FriendlyName" type="char_string" length="32"/>
+    <item fieldId="2" name="GroupKeySetID" type="int16u"/>
+    <item fieldId="3" name="GroupCAT" type="int16u"/>
+    <item fieldId="4" name="GroupCATVersion" type="int16u"/>
+    <item fieldId="5" name="GroupPermission" type="AccessControlEntryPrivilegeEnum"/>
+  </struct>
+
+  <struct name="DatastoreBindingEntry">
+    <cluster code="0x0752"/>
+    <item fieldId="0" name="ListID" type="int16u"/>
+    <!-- TODO: Need to support WrappedStructs when dealing with FabricScoped items -->
+    <!-- <item fieldId="1" name="Binding" type="TargetStruct"/> -->
+    <item fieldId="2" name="StatusEntry" type="DatastoreStatusEntry"/>
+  </struct>
+
+  <struct name="DatastoreGroupIDEntry">
+    <cluster code="0x0752"/>
+    <item fieldId="0" name="GroupID" type="group_id"/>
+    <item fieldId="1" name="StatusEntry" type="DatastoreStatusEntry"/>
+  </struct>
+
+  <struct name="DatastoreEndpointEntry">
+    <cluster code="0x0752"/>
+    <item fieldId="0" name="EndpointID" type="endpoint_no"/>
+    <item fieldId="1" name="NodeID" type="node_id"/>
+    <item fieldId="2" name="FriendlyName" type="char_string" length="32"/>
+    <item fieldId="3" name="StatusEntry" type="DatastoreStatusEntry"/>
+    <item fieldId="4" name="GroupIDList" type="DatastoreGroupIDEntry" array="true"/>
+    <item fieldId="5" name="BindingList" type="DatastoreBindingEntry" array="true"/>
+  </struct>
+
+  <struct name="DatastoreACLEntry">
+    <cluster code="0x0752"/>
+    <item fieldId="0" name="ListID" type="int16u"/>
+    <!-- TODO: Need to support WrappedStructs when dealing with FabricScoped items -->
+    <!-- <item fieldId="1" name="ACLEntry" type="AccessControlEntryStruct"/> -->
+    <item fieldId="2" name="StatusEntry" type="DatastoreStatusEntry"/>
+  </struct>
+
+  <struct name="DatastoreNodeInformationEntry" isFabricScoped="true">
+    <cluster code="0x0752"/>
+    <item fieldId="1" name="NodeID" type="node_id"/>
+    <item fieldId="2" name="FriendlyName" type="char_string" length="32"/>
+    <item fieldId="3" name="CommissioningStatusEntry" type="DatastoreStatusEntry"/>
+    <item fieldId="4" name="NodeKeySetList" type="DatastoreNodeKeyEntry" array="true"/>
+    <item fieldId="5" name="ACLList" type="DatastoreACLEntry" array="true"/>
+    <item fieldId="6" name="EndpointList" type="DatastoreEndpointEntry" array="true"/>
```

Comment:
I don't see the Revoked field being added (https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/11294/files)

Reply:
The Revoked field is in my PR: https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/11294

Reply:
Ok, we can submit it as a follow up PR.

Reply:
Group agreement: to be done in a follow up PR.